libs/Helpers/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# Light helpers library expected to be used by all other libraries
+#
+cmake_minimum_required(VERSION 3.21)
+
+project(Helpers
+    VERSION 0.1.0
+    LANGUAGES CXX)
+
+set(QT_NO_CREATE_VERSIONLESS_FUNCTIONS true)
+
+find_package(Qt6 6.3 COMPONENTS Quick Qml REQUIRED)
+qt6_standard_project_setup()
+
+add_library(${PROJECT_NAME} SHARED "")
+
+# Setup configuration type (Debug/Release)
+# Inspired by https://programmingrecluse.wordpress.com/2020/02/04/detect-debug-build-with-cmake/
+if(NOT CMAKE_CONFIGURATION_TYPES)
+    set(CMAKE_CONFIGURATION_TYPES Debug Release)
+elseif(NOT CMAKE_BUILD_TYPE)
+    message("Missing Build Type! Run cmake with:\n-DCMAKE_BUILD_TYPE=Debug|Release")
+endif()
+
+# Use by linker only
+set_property(GLOBAL PROPERTY DEBUG_CONFIGURATIONS Debug)
+
+set(BUILD_DEVELOPMENT ON CACHE BOOL "Activate developer specific features")
+
+# Setup BUILD_DEBUG and BUILD_REALEASE if single configuration builds
+if(CMAKE_BUILD_TYPE)
+    string(TOUPPER "${CMAKE_BUILD_TYPE}" _upper_build_type)
+    set(BUILD_${_upper_build_type} 1)
+    add_compile_definitions(DEVELOPMENT_BUILD=true)
```

Comment:
Why is that? `DEVELOPMENT_BUILD` is always true for single configuration builds, looks wrong :P

Reply:
Fixed with the latest changes. Converted into a CMake cache option to be customized with ccmake.